### ISSUE TITLE
Error: EMFILE: too many open files

### ISSUE THREAD

**Issue by viperfx** [quirrel-dev/quirrel#48]
When I run the quirrel command to start the server, I get lots of errors similar to `Error: EMFILE: too many open files`

```
Error: EMFILE: too many open files, open 'widget/node_modules/wrap-ansi/node_modules/ansi-regex/index.js'
This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
Error: EMFILE: too many open files, open 'widget/node_modules/wrap-ansi/node_modules/strip-ansi/index.d.ts'
This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
Error: EMFILE: too many open files, open 'widget/node_modules/wrap-ansi/node_modules/strip-ansi/index.js'
This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
Error: EMFILE: too many open files, open 'widget/node_modules/yargs-parser/node_modules/camelcase/index.d.ts'
This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
Error: EMFILE: too many open files, open 'widget/node_modules/yargs-parser/node_modules/camelcase/index.js'
....
```

I am using WSL2 with redwood client libraries. I am running the command with `yarn run quirrel`

**Comment by Skn0tt** [comment]
Apparently we don‘t ignore enough files. Will take a look it this weekend.

**Comment by Skn0tt** [comment]
#51 should fix this. @viperfx Could you try out `1.0.2-fix-48`? (`npm install quirrel@danger`)

**Comment by Skn0tt** [comment]
Published in 1.0.3.
